invenio_oauthclient/contrib/cern_openid.py

```diff
@@ -210,6 +210,7 @@ def extend_identity(identity, roles):
 
 def disconnect_identity(identity):
     """Disconnect identity from CERN groups."""
+    session.pop("cern_resource", None)
```

Comment:
I find the name `disconnect_identity` quite misleading... but probably for another time

Reply:
`disconnect_identity` the disconnect term is used in a more wide spectrum in the repo, not just in this case, thus I let it be. I agree that it might not be the most self explained.

Reply:
I guess it was used because until now was part of the "disconnect" workflow :)